src/qml/FeatureForm.qml

```diff
@@ -262,13 +262,9 @@ Page {
           anchors { left: parent.left; right: parent.right }
 
           //disable widget if the form is in ReadOnly mode, or if it's an RelationEditor widget in an embedded form
-          property bool isEnabled: AttributeAllowEdit && !!AttributeEditable && (
-                                     form.state !== 'ReadOnly'
-                                     || EditorWidget === 'RelationEditor'
-                                     || EditorWidget === 'ValueRelation'
-                                     || EditorWidget === 'ExternalResource'
-                                     )
-          property bool readOnly: form.state === 'ReadOnly' || embedded && EditorWidget === 'RelationEditor'
+          property bool isEnabled: AttributeAllowEdit && !!AttributeEditable &&
+                                   form.state !== 'ReadOnly' && !( embedded && EditorWidget === 'RelationEditor' )
+                                   && !(form.model.featureModel.modelMode == FeatureModel.MultiFeatureModel && Type === 'relation')
```

Comment:
Personally I prefer such cluster of boolean conditions (more than 2) to be all on new line, so it is easy to read them one by one. Or at least move the operator on the same side.
```suggestion
          property bool isEnabled: AttributeAllowEdit 
              && !!AttributeEditable 
              && form.state !== 'ReadOnly' 
              && !(embedded && EditorWidget === 'RelationEditor')
              && !(form.model.featureModel.modelMode == FeatureModel.MultiFeatureModel && Type === 'relation')
```

Reply:
Thanks. I did that in my last (squashed) commit. 
Additionally, I used one line for the relation related things ( `EditorWidget === 'RelationEditor'` and `Type === 'relation'` is the same since relation have only this widget) and some description.

Reply:
Cool!